var/spack/repos/builtin/packages/py-clean-text/package.py

```diff
@@ -13,9 +13,13 @@ class PyCleanText(PythonPackage):
 
     pypi = "clean-text/clean-text-0.5.0.tar.gz"
 
+    version("0.6.0", sha256="8374b385fc2a26e06383f62aed076fa6be115e5832239e2a7fd8b344fa8d2ab2")
     version("0.5.0", sha256="e525951bef0c8b72e03c987fdac2c475b61d7debf7a8834366fd75716179b6e1")
 
     depends_on("python@3.6:", type=("build", "run"))
+    depends_on("python@3.7:", type=("build", "run"), when="@0.6.0:")
+    depends_on("py-poetry@0.12:", type="build")
     depends_on("py-setuptools", type="build")
     depends_on("py-emoji", type=("build", "run"))
-    depends_on("py-ftfy@6.0:6.999", type=("build", "run"))
+    depends_on("py-emoji@1.0:1", type=("build", "run"), when="@0.6.0:")
```

Comment:
Can you please switch `type` and `when`? Also This can be shortened to
```suggestion
    depends_on("py-emoji@1", when="@0.6:", type=("build", "run"))
```